src/server/styles/index.jsx

```diff
@@ -16,7 +17,7 @@ export const getStyleTag = (sheet, isAmp = false) => {
 
   return (
     // eslint-disable-next-line react/no-danger
-    <style amp-custom="" dangerouslySetInnerHTML={{ __html: inlineCss }} />
+    <style amp-custom="" dangerouslySetInnerHTML={{ __html: xss(inlineCss) }} />
```

Comment:
If we want to render global styles I think style-components provides a way to do this https://www.styled-components.com/docs/api#createglobalstyle

Reply:
* Using a style tag without dSIH escapes things like urls in CSS.  
* Global styles doesn't seem to work with `renderToString`. I haven't been able to add a custom attribute (`amp-custom`) to the global style either.

So far, using `dSIH` here seems to be the "easiest" implementation.